### DISCUSSION TITLE
error useLocalStorageState

### DISCUSSION THREAD

**Discussion by donacrisante** [neuefische/questions#666] [Web] [answered]
Hello! Since I installed useLocalStorageState, I get this error that I can't get fixed. Can anyone help me please?

![Bildschirm­foto 2023-08-14 um 11 46 30](https://github.com/neuefische/questions/assets/128093782/df2818b9-a527-4063-a425-2373453a0d69)

**Comment by Kheirah** [comment]
Please leave a link to your latest pull request =)

**Reply by donacrisante** [reply]
Hi, here my PR:
https://github.com/donacrisante/Capstone-project/pull/10

**Comment by Kheirah** [comment, marked as answer]
Try downgrading to the version 18.0.0-0 and see if that fixes the error =)

**Reply by ninaassmann** [reply]
had the same issue, deleted the node_modules and package.lock.json
installed npm i use-local-storage-state@18
npm i after that

and it worked 👍
